templates/tools/dockerfile/test/csharp_stretch_x64/Dockerfile.template

```diff
@@ -14,6 +14,9 @@
   # See the License for the specific language governing permissions and
   # limitations under the License.
   
+  # TODO(https://github.com/grpc/grpc/issues/19083): Remove dependency on Python
+  # 2.
```

Comment:
Since this string appears so many time, should we compress it into one line? Something like:

```
 # TODO(https://github.com/grpc/grpc/issues/19083): Migrate to Python 3
```

Reply:
I don't really see the value in this. What does this gain us?

Reply:
Remove a line `# 2.`. It's optional.